applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -288,6 +289,26 @@ static int random_static_addr_cfg(void)
 	return 0;
 }
 
+static int random_static_addr_set(void)
```

Comment:
Suggest: Have a comment on this function as well, like the one you have besides the Kconfig.